DigitalLearningSolutions.Web/Views/SuperAdmin/Shared/_SideNavMenu.cshtml

```diff
@@ -0,0 +1,34 @@
+@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewComponents
+<h2 class="side-nav__heading">System</h2>
+<ul class="nhsuk-list side-nav__list">
+  <vc:side-menu-link asp-action="Index"
+                     asp-controller="Faqs"
+                     link-text="Configuration"
+                     is-current-page="@(Model.CurrentPage == SuperAdminSystemPage.Configuration)" />
+
+  <vc:side-menu-link asp-action="Index"
+                     asp-controller="Faqs"
+                     link-text="Manage Lists"
+                     is-current-page="@(Model.CurrentPage == SuperAdminSystemPage.ManageLists)" />
+
+  <vc:side-menu-link asp-action="Index"
```

Comment:
Ah just spotted, the wireframe is out of date. This item can be removed as `Landing Content` is not being refactored here.